Makefile

```diff
@@ -79,7 +79,11 @@ else
     SECCOMP=no
 endif
 # go build common flags
-BUILDFLAGS := -buildmode=pie
+ifndef STATIC
```

Comment:
maybe you should change this condition, I think the following would be easier to understand

```make
ifdef STATIC
	LDFLAGS := -extldflags '-static'
else
	BUILDFLAGS := -buildmode=pie
endif
```

Reply:
> maybe you should change this condition, I think the following would be easier to understand
> 
> ```makefile
> ifdef STATIC
> 	LDFLAGS := -extldflags '-static'
> else
> 	BUILDFLAGS := -buildmode=pie
> endif
> ```

Updated.